operators/dep-bundle-operator/Makefile

```diff
@@ -142,7 +145,8 @@ run: manifests generate fmt vet ## Run a controller from your host.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: docker-build
 docker-build: ## Build docker image with the manager.
-	$(CONTAINER_TOOL) build -t ${IMG} .
+	$(CONTAINER_TOOL) build --build-arg "HOSTPLATFORM=$(HOST_GOOS)/$(HOST_GOARCH)" \
+	  --build-arg "TARGETARCH=$(HOST_GOARCH)" --build-arg "TARGETOS=$(HOST_GOOS)" -t ${IMG} .
```

Comment:
Curious: Any reason we're not using these operator targets in the scripts? (Though I'll say the command visibility in the scripts is nice.)

Reply:
Mostly I'm just a fan of less re-direction. Especially in this case, you'd need to adjust some variables that the Make target uses, so you need to understand how it works in the script, in the Makefile *and* in the Dockerfile.

The Make targets were adjusted and I intended for them to primarily work in a single-arch setup, when you just need an image "locally".